test/check-application

```diff
@@ -29,23 +29,54 @@ class TestNavigator(testlib.MachineCase):
 
         self.restore_dir("/home/admin")
 
+        # Set browser timezone to UTC as the displayedtime in respecting the browser timezone
+        os.environ['TZ'] = 'UTC'
```

Comment:
Would be nice to restore it at the end of the test, to avoid unexpected side effects on other tests. Or just move it to `setUpClass()`, so that it applies to all tests -- that's maybe better?